apps/web/app/admin/(items)/_components/AddCategoryForm.tsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+import { Button, Input, notify } from "@ui/index";
+import { useForm } from "react-hook-form";
+import { postNewCategory } from "@/api/meetings";
+
+export default function AddCategoryForm(): JSX.Element {
+  const { register, handleSubmit } = useForm();
+  const queryClient = useQueryClient();
+
+  const mutation = useMutation({
+    mutationFn: async (payload: Record<string, string>) => {
+      return await postNewCategory(payload);
+    },
+    onSuccess: async () => {
+      notify({ type: "success", message: "카테고리가 추가되었습니다!" });
+      await queryClient.invalidateQueries({ queryKey: ["categories"] });
+    },
+    onError: () => {
+      notify({ type: "error", message: "요청에 실패했습니다." });
+    },
+  });
+
+  const handleSubmitForm = handleSubmit((data) => {
+    const payload = {
+      ...data,
+      itemType: "room",
+    };
+
+    mutation.mutate(payload);
```

Comment:
```suggestion
    mutation.mutate(payload);
```
체이닝이 아닌 구조분해할당으로 뽑아쓰는건 어때?
```ts
const { mutate: mutate for what? } = useMutation()
```
이런식으로 어떤 mutate인지 네이밍해서